templates/quickstart-database-for-atlassian-services.yaml

```diff
@@ -255,3 +281,19 @@ Outputs:
     Condition: UseDatabaseEncryption
     Description: The alias of the encryption key created for RDS
     Value: !If [UseAurora, !GetAtt AuroraDatabase.Outputs.RDSEncryptionKey, !GetAtt PostgresDatabase.Outputs.RDSEncryptionKey]
+  PostgresTemplateURL:
+    Condition: UsePostgres
+    Description: The URL used to source the PostgreSQL Stack template
+    Value:
+      !Sub
+      - https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}submodules/quickstart-atlassian-services/templates/quickstart-postgres-for-atlassian-services.yaml
+      - S3Bucket: !If [UsingDefaultBucket, !Sub '${QSS3BucketName}-${AWS::Region}', !Ref QSS3BucketName]
+        S3Region: !If [UsingDefaultBucket, !Ref 'AWS::Region', !Ref QSS3BucketRegion]
+  AuroraTemplateURL:
+    Condition: UseAurora
+    Description: The URL used to source the Aurora Stack template
+    Value:
+      !Sub
+      - https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}submodules/quickstart-atlassian-services/submodules/quickstart-amazon-aurora/templates/aurora_postgres.template.yaml
+      - S3Bucket: !If [UsingDefaultBucket, !Sub '${QSS3BucketName}-${AWS::Region}', !Ref QSS3BucketName]
+        S3Region: !If [UsingDefaultBucket, !Ref 'AWS::Region', !Ref QSS3BucketRegion]
```

Comment:
Added these for testing purposes. I was planning on removing them but now I'm thinking is there any harm in keeping them? Now that we are moving to regional buckets its might be nice to be able to confirm that the templates are being sourced from where you'd expect them. This info is also useful for us, when debugging purposes stack issues.